ci-operator/step-registry/aws-load-balancer/post-install-rosa/aws-load-balancer-post-install-rosa-commands.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+AWSCRED="${CLUSTER_PROFILE_DIR}/.awscred"
+REGION="${LEASED_RESOURCE}"
+E2E_INPUT_DIR="${SHARED_DIR}"
+E2E_WAFV2_WEB_ACL_NAME="echoserver-acl"
+
+echo "=> configuring aws"
+if [ -f "${AWSCRED}" ]; then
+  export AWS_SHARED_CREDENTIALS_FILE="${AWSCRED}"
+  export AWS_DEFAULT_REGION="${REGION}"
+else
+  echo "Did not find compatible cloud provider cluster_profile"; exit 1
+fi
+
+if [ -f ${E2E_INPUT_DIR}/wafv2-webacl ]; then
+    E2E_WAFV2_WEB_ACL_ARN="$(cat ${E2E_INPUT_DIR}/wafv2-webacl)"
+    E2E_WAFV2_WEB_ACL_ID="${E2E_WAFV2_WEB_ACL_ARN##*/}"
+    echo "=> deleting e2e wafv2 web acl: ${E2E_WAFV2_WEB_ACL_ARN}"
+    LOCK_TOKEN=$(aws wafv2 get-web-acl --name "${E2E_WAFV2_WEB_ACL_NAME}" --id "${E2E_WAFV2_WEB_ACL_ID}" --scope=REGIONAL --output json | jq -r .LockToken)
+    aws wafv2 delete-web-acl --name "${E2E_WAFV2_WEB_ACL_NAME}" --id "${E2E_WAFV2_WEB_ACL_ID}" --scope=REGIONAL --lock-token "${LOCK_TOKEN}"
+else
+    echo "=> nothing to do for e2e wafv2 web acl"
+fi
+
+if [ -f ${E2E_INPUT_DIR}/waf-webacl ]; then
+    # it's possible to create webacls with duplicate name using wafregional,
+    # we have to take this case into account
+    for id in $(cat ${E2E_INPUT_DIR}/waf-webacl); do
```

Comment:
You used `cat ${SHARED_DIR}/waf-webacl | head -1` in `ci-operator/config/openshift/aws-load-balancer-operator/openshift-aws-load-balancer-operator-main.yaml`.  Do you need `head -1` here too?  (Alternatively, you could do the `head -1` when writing the file in `ci-operator/step-registry/aws-load-balancer/pre-install-rosa/aws-load-balancer-pre-install-rosa-commands.sh`.)

Reply:
Here I'd like to make sure all the web acls for the e2e test are removed. I noticed (by running the rehearsals) that for the legacy waf (wafregional) you can create duplicate webacls (impossible with wafv2 btw).
In case something goes wrong with the CI or with this step and we get create multiple webacls, I want 1) the e2e test to get only 1 webacl (`head -1`) because one is enough, 2) the post install to delete all the created e2e webacls.

Reply:
BTW, you might want to quote `${E2E_INPUT_DIR}/waf-webacl` for the unlikely case that `${E2E_INPUT_DIR}` has spaces or other strange characters.

Reply:
Done.